defs/go.bzl

```diff
@@ -54,9 +53,8 @@ def _compute_genrule_command(ctx, go_stdlib):
   return '\n'.join(cmd)
 
 def _go_genrule_impl(ctx):
-  go = go_context(ctx)
-
-  all_srcs = depset(go.stdlib.files)
+  go_stdlib = get_source(ctx.attr._stdlib).stdlib
```

Comment:
This change should not be needed, now you declare with go_rule you should be able to build a go_context

Reply:
ah, I think I'm finally starting to understand. this is looking much cleaner now.